### PR TITLE
capitalize the "I" in Iviewer

### PR DESCRIPTION
**Pull request by lunson**
Fixes the issue in https://trello.com/c/LF6cK3DW/101-omeroiviewer-instead-of-omeroiviewer

### CODE REVIEW

**Comment by hflynn** [comment]
it's the other way around @lunson - it's capitalized now by the styling and should be lowercase

**Comment by lunson** [comment]
@hflynn Okay, I just realized that on Firefox it's lowercase and on Chrome it's not, so I thought it was the other way around. Will fix.

**Comment by hflynn** [comment]
Ah, i hadn't realised that the browser made a difference, I'll be sure to check not just Safari in future!

**Comment by jburel** [comment]
I was using Chrome

**Comment by hflynn** [comment]
Looks fine on staging in Chrome and Firefox now. In Safari I can't actually see the difference between 'I' and 'i' on my screen which would explain how i missed this in the first place!

File: omero/analyze/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h2 class="header-blue">Analyze your data in OMERO</h2>
                 <p>Draw ROIs across multiple dimensions and measure pixel
                     intensity. </p>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/measurement-tool.html" target="_blank">View Insight user guide</a>
-                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=FQD540TTwFg" target="_blank">Watch OMERO.iviewer movie</a>
+                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=FQD540TTwFg" target="_blank">Watch OMERO.<span style="text-transform: lowercase;">i</span>viewer movie</a>
```
Comment (joshmoore) [inline omero/analyze/index.html]: That's one verbose "lowercase"! Would a CSS class `#productname` (or similar) be possible for usability?

Comment (lunson) [inline omero/analyze/index.html]: It will still be "verbose" though because this is a one-off that overrides a style applied to all the buttons.

**Comment by hflynn** [comment]
@jburel @joshmoore can this be merged now?

**Comment by jburel** [comment]
Merging so we can move forward with the other changes

**Pull request merged**